.github/workflows/release.yml

```diff
@@ -7,62 +7,48 @@ on:
     - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
 
 jobs:
-  lintTestBuild:
+  test:
     name: Lint, Test, Build
     runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.extract-tag.outputs.VERSION }}
+      vversion: ${{ steps.extract-tag.outputs.VVERSION }}
+      is_prerelease: ${{ steps.is_prerelease.outputs.IS_PRERELEASE }}
     steps:
-      - name: Checkout
+      -
+        name: Checkout
         uses: actions/checkout@v2
-      - name: Setup Go
+      -
+        name: Setup Go
         uses: actions/setup-go@v2
         with:
           go-version: '^1.15.6'
```

Comment:
Will this use 1.16? Do we want to be backward compatible with older versions of Go?